lib/cisco_node_utils/node.rb

```diff
@@ -260,7 +265,7 @@ def product_id
           # No support for structured output for this command yet
           output = get(command:     'show inventory',
                        data_format: :cli)
-          return /NAME: "Rack 0".*\nPID: (\S+)/.match(output)[1]
+          return /NAME: .*\nPID: (\S+)/.match(output)[1]
```

Comment:
Made the regex a little less specific, since we had a case where the output did not include the "Rack 0" part.